config/utils/buildGrmVoteApiUri.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  GrmVoteType,
+  ID,
+  RhDocumentType,
+} from "../types/root_types";
+import { buildApiUri } from "./buildApiUri";
+import { isNullOrUndefined } from "./nullchecks";
+
+type Args = {
+  commentPayload?: {
+    commentID?: string;
+    commentType?: string;
+    replyID?: string;
+    threadID?: string;
+  };
+  documentType: RhDocumentType;
+  documentID: ID;
+  voteType: GrmVoteType;
+};
+
+export function buildGrmVoteApiUri({
```

Comment:
Can we update the `FLAG_GRM_CONTENT` in `api.js` to utilize this function?

Reply:
I dont want to bundle this with Flag api since they are separate models with separate endpoints. I can move this to a separate file like Votes later if necessary.